active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="ConnectionErrorNotification"
+        Title="Error while saving"
+        Message="Lorem ipsum long message">
+        <controls:InfoBar.ActionButton>
+            <HyperlinkButton
+                Content="www.microsoft.com" 
+                NavigateUri="http://www.microsoft.com"/>
+        <controls:InfoBar.ActionButton/>
+    </controls:InfoBar>
+</StackPanel>
+```
+![A mockup of an InfoBar with a single line message and hyperlink](images/Default_MessageHyperlink.png)
+![A mockup of an InfoBar with a message expanding multiple lines and a hyperlink](images/Docked_MultiLineIconTitleHyperlink.png)
+
+## Custom content
+Content can be added to an InfoBar using the Content property. It will appear below the message and above the action or hyperlink buttons if defined and multiline. The InfoBar will expand to fit the content defined.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="TermsAndConditionsNotification"
+        Title="Backup in progress"  
+        Message="Your documents are being saved to the cloud">
+        <controls:InfoBar.Content>
+            <ProgressBar IsIndeterminate="True"/>
+        </controls:InfoBar.Content>
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar in its default state with an indeterminate progress bar](images/Default_CustomContent.gif)
+
+
+## Content wrapping
+By default, the text set in the Message property will wrap vertically in the control underneath the other visual components.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="DefaultErrorNotification"
+        Severity="Error"
+        Title="Message Title"  
+        Message="This message is very long, so long in fact it needs to wrap to a second line in the notification">
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of a multiline message InfoBar](images/Docked_MultiLineIconTitle.png)
+![A mockup of a multiline title and message InfoBar](images/Docked_MultiLineLongTitle.png)
+
+## Canceling and deferring close
+The Closing event can be used to cancel and/or defer the close of an InfoBar. This can be used to keep the InfoBar open or allow time for an action or custom animation to occur. When the closing of an InfoBar is canceled, IsOpen will go back to true, however, it will stay false during the deferral. A programmatic close can also be canceled.
+
+XAML
+```xml
+<controls:InfoBar x:Name="UpdateAvailable"
+    Title="Update Available"
+    Message="Please close this tip to apply required security updates to this application"
+    Closing="InfoBar_Closing">
+</controls:InfoBar>
+```
+C#
+```C#
+public void InfoBar_Closing(InfoBar sender, InfoBarClosingEventArgs args)
+{
+    if (args.Reason == InfoBarCloseReason.CloseButton) {
+        // if scenario failed
+        args.Cancel = true;
+    }
+    
+}
+```
+
+# Inputs and Accessibility
```

Comment:
The public docs don't usually go into this much detail, but it's great detail. This can be moved to an Appendix of this spec.